theme.json

```diff
@@ -0,0 +1,413 @@
+          "fontWeight": "var(--wp--custom--heading--5--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--5--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--5--letter-spacing)"
+        }
+      },
+      "h2": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--4--font-size)",
+          "lineHeight": "var(--wp--custom--heading--4--line-height)",
+          "fontWeight": "var(--wp--custom--heading--4--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--4--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--4--letter-spacing)"
+        }
+      },
+      "h3": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--3--font-size)",
+          "lineHeight": "var(--wp--custom--heading--3--line-height)",
+          "fontWeight": "var(--wp--custom--heading--3--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--3--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--3--letter-spacing)"
+        }
+      },
+      "h4": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--2--font-size)",
+          "lineHeight": "var(--wp--custom--heading--2--line-height)",
+          "fontWeight": "var(--wp--custom--heading--2--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--2--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--2--letter-spacing)"
+        }
+      },
+      "h5": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--1--font-size)",
+          "lineHeight": "var(--wp--custom--heading--1--line-height)",
+          "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--1--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+        }
+      },
+      "h6": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--1--font-size)",
+          "lineHeight": "var(--wp--custom--heading--1--line-height)",
+          "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--1--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+        }
+      },
+      "button": {
+        "color": {
+          "background": "var(--wp--custom--button--primary--default)",
+          "text": "var(--wp--custom--button--primary--label)"
+        },
+        "typography": {
+          "fontSize": "clamp(1.00rem, calc(1.16rem + -0.15vw), 1.13rem)",
+          "lineHeight": "var(--wp--custom--body--2--line-height)",
+          "fontWeight": "var(--wp--custom--body--2--font-weight)",
+          "fontFamily": "var(--wp--custom--body--2--font-family)"
+        },
+        "spacing": {
+          "padding": {
+            "top": "clamp(0.50rem, calc(0.81rem + -0.30vw), 0.75rem)",
```

Comment:
Same here. This is where you start seeing the gaps between the way I implemented spacing and the way you probably shoudl implement it. These should probably be references to button-specific padding tokens.

Reply:
I'll move them to be button specific padding tokens.

Reply:
I'm leaning towards putting these under [the existing space section referenced here](https://github.com/Automattic/vip-design-system-bridge/pull/31#discussion_r1323974258). This way, we do a mixed approach to spacing. If a developer wants to reference the `space.responsive.1` they can, or else they can move it to their own component specific token set as well. 

I'd lean towards keeping the token set and the `theme.json` minimal as much as possible for this iteration